### ISSUE TITLE
Global PHP configuration options

### ISSUE THREAD

**Issue by smithandre** [phpbrew/phpbrew#695]
PHPBrew is great, but it does lack the ability to configure certain php settings globally, and have them apply to each version one installs. 

Initially I was thinking of having a central ini file containing the options, and then just using a symlink to reference it. However, different versions of PHP could have different options in the config, which is why I think that won't work well, as one will "lose" the differences in the configuration between versions (if any)

So... my thinking is to build on the idea of having a .yml in which one can specify core options like memory limits and log paths, as well as extension configuration. When one builds a new PHP, phpbrew should then use these options and set them in the ini of the specific version you are busy installing.

**Comment by c9s** [comment]
Thanks, there is already a global configuration, it should be documented in wiki.

**Comment by c9s** [comment]
See this page for more details https://github.com/phpbrew/phpbrew/wiki/Setting-up-Configuration

**Comment by c9s** [comment]
I think what you mentioned could be extended from the current configuration file.

**Comment by smithandre** [comment]
I agree. It will be better to extend from existing functionality rather than to build something separate.
